samples/docker-domain/README.md

```diff
@@ -0,0 +1,50 @@
+Example Image with a  WLS Domain
+================================
+This Dockerfile extends the Oracle WebLogic image by creating a sample WLS 12.2.1.3 domain and cluster from a WebLogic Deploy Tool model.
+
+Utility scripts are copied into the image, enabling users to plug Node Manager automatically into the Administration Server running on another container.
```

Comment:
This is more a question, since I'm hoping to see them below, but... What are these scripts and what exactly do they do?